tests/unit/test_general_helper.py

```diff
@@ -49,9 +49,9 @@ def test_get_themerr_json_path(movies):
                             'DataItems') in themerr_json_path
 
 
-def test_get_themerr_json_data(movies):
+def test_get_themerr_json_data(section):
     test_items = [
-        movies.all()[0]
+        section.all()[0]
     ]
 
     for item in test_items:
```

Comment:
Test failing because, no theme is being added by Themerr for `TV Shows` or `TV Shows-tvdb`.